ProofPoint/proofpoint-tap/ingest/parser.yml

```diff
@@ -94,7 +101,7 @@ stages:
                   {% elif threat.get("threatType", "").lower() == "url" %}
                   "type": "domain-name",
                   "url": {
-                    "domain": "{{threat.threat}}"
+                    "original": "{{threat.threat}}"
```

Comment:
Can you add the `url.domain` with the same value in order to do a smooth transition? The field `threat.enrichments.indicator.url.domain` is currently used.